src/components/Footer/Footer.jsx

```diff
@@ -35,13 +35,24 @@ const Footer = () => {
 
       <div className={styles.socialContent}>
         <div className={styles.socialLinks}>
-          <FacebookIcon className={styles.socialLink} />
-          <PinterestIcon className={styles.socialLink}/>
-          <TwitterIcon className={styles.socialLink}/>
+          <img
+            src={FacebookIcon}
+            className={styles.socialLink}
+            alt="facebook"
+          />
+          <img
+            src={PinterestIcon}
+            className={styles.socialLink}
+            alt="Pinterest"
+          />
+          <img src={TwitterIcon} className={styles.socialLink} alt="twitter" />
```

Comment:
This needs to remain the same to apply the hover style
```suggestion
import { ReactComponent as FacebookIcon } from "../../assets/icon-facebook.svg";
import { ReactComponent as PinterestIcon } from "../../assets/icon-pinterest.svg";
import { ReactComponent as TwitterIcon } from "../../assets/icon-twitter.svg";